### PR TITLE
add update hound action

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: .github/workflows/hound.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            experimental-features = nix-command flakes
+      - name: Setup
+        id: setup
+        run: |
+          echo "title=hound: update repos $(date -u +"%Y-%m-%d")" >> $GITHUB_OUTPUT
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+      - name: Update hound repos
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_FOR_UPDATES }}
```
Comment (Mic92) [inline .github/workflows/hound.yml:27]: Token should be fine, potentially not required, you can also trigger CI using bors and our hercules one should be triggered also with the github bot provided tokens.

Comment (zowoq) [reply .github/workflows/hound.yml:27]: This is to avoid the repo update script being rate limited. 

Default token would work, I'll change it for both.

Comment (Mic92) [reply .github/workflows/hound.yml]: Upside of that is that we can also see if user manually triggered PR creation this way.

**Comment by Mic92** [comment]
bors merge

**Pull request merged**